.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+
+language: ruby
+
+services:
+  - docker
+
+before_install:
+- docker pull debian:sid-slim
+
+script:
+- docker run -v $PWD:/mnt/kaidan debian:sid-slim /bin/bash -c "apt update; apt install software-properties-common dirmngr gnupg ca-certificates git -y; apt-key adv --keyserver hkp://pgp.mit.edu --recv-key "210EB0BCA70439F0"; echo deb https://raw.githubusercontent.com/kaidanim/packages/master sid main >> /etc/apt/sources.list; apt-get update; apt-get install cmake build-essential zlib1g-dev libgloox-dev libqt5quick5 libqt5quickcontrols2-5 libqt5quickwidgets5 libqt5qml5 libqt5gui5 libqt5core5a qtdeclarative5-dev qttools5-dev qt5-default qtquickcontrols2-5-dev kirigami2-dev -y; cd /mnt/kaidan; mkdir build; cd build; cmake .. -DI18N=1; make -j$(nproc)"
```

Comment:
A space is missing at the beginning of the line.
Please separate every command into a new line (starting with ` - `).

Reply:
Alternatively, you can also create a shell script and just execute that.

Reply:
>  Please separate every command into a new line (starting with -).
It's only one

Reply:
Then maybe something like `bash -c $(cat util/travis.sh)`?

Reply:
Or just `bash -c ./util/travis.sh`